cpu/Makefile.include.gnu

```diff
@@ -9,6 +9,11 @@ endif
 export AS = $(PREFIX)as
 export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
-export OBJCOPY = $(PREFIX)objcopy
+ifneq ("$(wildcard $(PREFIX)objcopy)","")
```

Comment:
Change this to something like:
```make
ifeq (,$(OBJCOPY))
ifneq (,$(shell command -v $(PREFIX)objcopy 2>/dev/null))
export OBJCOPY = $(PREFIX)objcopy
else
ifneq (,$(shell command -v gobjcopy 2>/dev/null))
export OBJCOPY = gobjcopy
else
export OBJCOPY = objcopy
endif
endif
endif
```
(untested)
I reversed the condition to use `gobjcopy` if it is found, and fall back to `objcopy`.
This is Makefile.include.gnu we're working in and some systems prefix the GNU tools with a `g` to distinguish from the vendor tools (Solaris comes to mind as one example of this).

Why did you need to add `-O ihex` on the `gobjcopy` but not on `objcopy`? `gobjcopy` should be a program with the same behaviour as the normal `objcopy`, only a different name.

Also, there's no reason to use double quotes in Makefiles because `make` does not handle any kind of string quoting.